src/.env

```diff
@@ -1,4 +1,4 @@
 API_USER=
 API_PASSWORD=
-API_URL=https://api.ed-fi.org/v5.3/api/data/v3/ed-fi
+API_URL=https://api.ed-fi.org/v5.3/api/data/v3/
 API_URL_TOKEN=https://api.ed-fi.org/v5.3/api/oauth/token
```

Comment:
Please remove this file. Although it does not have secure credentials in it, anyone else getting the repository will end up modifying to add their client credentials, and then they might accidentally save it. Best to add `.env` to the `.gitignore` file.